graphql/src/graphql_schema.ml

```diff
@@ -1079,7 +1089,9 @@ end
           | Some field ->
               resolve_field ctx src query_field field
           | None ->
-              Io.ok ((alias_or_name query_field, `Null), [])
+              let error_message = Printf.sprintf "Cannot query field '%s' on type '%s'" query_field.name obj.name in
```

Comment:
To align with the wording of the spec, I'd suggest `"Field '%s' is not defined on type '%s'"`.